src/wcs1_1/src/main/java/org/geoserver/wcs/response/CoveragesTransformer.java

```diff
@@ -124,7 +124,7 @@ void handleCoverage(CoverageInfo ci) throws Exception {
             start("wcs:Coverage");
             element("ows:Title", ci.getTitle());
             element("ows:Abstract", ci.getDescription());
-            element("ows:Identifier", ci.getName());
+            element("ows:Identifier", ci.prefixedName());
```

Comment:
yep this looks appropriate, I assume this is how WCS 2 0 was functioning as well.